### PR TITLE
[Merged by Bors] - TY-2847 port tooling crate to de conventions

### PR DESCRIPTION
**Pull request by janpetschexain**
**References**

- [TY-2847]

**Summary**

- the first commit handles renaming
- the following commits fix a lint each

[TY-2847]: https://xainag.atlassian.net/browse/TY-2847?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: discovery_engine_core/tooling/bin/clean_sources.rs [hunk 1]
```diff
@@ -104,10 +120,10 @@ fn clean_sources(in_file: impl AsRef<Path>, out_file: impl AsRef<Path>) -> Resul
             bail!("missing domain field at {:?}", record.position());
         };
         if let Some(name) = record.get(0) {
-            if !name.is_empty() {
-                writer.write_byte_record(&[name, domain].as_ref().into())?;
-            } else {
+            if name.is_empty() {
```
Comment (acrrd) [inline discovery_engine_core/tooling/bin/clean_sources.rs:123]: This is an arbitrary change that is out of scope.

Comment (janpetschexain) [reply discovery_engine_core/tooling/bin/clean_sources.rs:123]: it's not arbitrary, but flagged by `clippy:pedantic`

**Comment by janpetschexain** [comment]
bors merge
